src/main/java/org/primefaces/extensions/component/letteravatar/LetterAvatarRenderer.java

```diff
@@ -113,29 +117,29 @@ public static int hue(String str) {
         return Math.abs(hash(str) % 360);
     }
 
-    protected String styleDiv(String size, String color, String backgroundColor, boolean rounded) {
+    protected String styleContainer(String size, String color, String backgroundColor, boolean rounded) {
         final Map<String, String> map = new LinkedHashMap<>(8);
         map.put("color", color);
         map.put("background-color", backgroundColor);
         if (rounded) {
             map.put("border-radius", "50%");
         }
         map.put("height", size);
-        map.put("text-align", "center");
         map.put("width", size);
         return toStyle(map);
     }
 
-    protected String styleSpan(String size) {
+    protected String styleInitials(String size) {
         final Map<String, String> map = new LinkedHashMap<>(8);
         map.put("font-size", "calc(" + size + " / 2)"); // 50% of parent
-        map.put("line-height", "1");
-        map.put("position", "relative");
         map.put("top", "calc(" + size + " / 4)"); // 25% of parent
-        map.put("mix-blend-mode", "difference");
         return toStyle(map);
```

Comment:
I thought this mixed-blend-mode make the letters contrast to the background?  So that way white letters not on a really light blue background making them impossible to read?

Reply:
I don't know. As it was fixed, I put it in the CSS file.